### ISSUE TITLE
Play Commands are not working

### ISSUE THREAD

**Issue by yogesh-oyo** [playframework/play1#1122]
Play Commands are not working e.g play run, play dependencies, and showing static response for everything.

Play Version (1.4.4)

API (Java)

Operating System (MacOS 10.11)

JDK "1.8.0_92"

Library Dependencies

play
** Earlier I added mongo dependency then removed it.

Expected Behavior

Resolve Dependencies stated in dependencies.yml

Actual Behavior

Showing static response for every play command e.g. play run, play dependencies inside repo.

~ _ _
~ _ __ | | __ _ _ | |
~ | ' | |/ ' | || ||
~ | /||_|__ ()
~ || |__/
~
~ play! 1.4.x-localbuild, https://www.playframework.com
~
~ Oops,
~ Module not found: /opt/play-1.4.4/modules/mongo

Reproducible Test Case

I don't have exact steps.

**
When I created a new repository, it worked fine.

**Comment by xael-fry** [comment]
Hi,  Now in Play 1.4.4 play dependencies do a sync by defaut (you can use --nosync to avoid it),
So you must have a problem to resolve the mongo module, so you have to declare it in dependencies.yml

**Comment by yogesh-oyo** [comment]
@xael-fry Thanks for quick response.

I tried 

play dependencies --nosync
play run --nosync

It is showing same static response.

**Comment by xael-fry** [comment]
@yogesh-oyo How did you declare mongo in dependencies.yml?

**Comment by yogesh-oyo** [comment]
# Application dependencies

require:
    - play
    - play -> redis 0.3
    - play -> mongo 1.3
    - play -> accesslog 1.2

**Comment by xael-fry** [comment]
No problem to resolve mongo 1.3, do you have a folder mongo-1.3 in /modules in your app? So that is the content of your application modules directory ?

**Comment by yogesh-oyo** [comment]
Yes, I have. I think it is problem with this repo only, in another repo all things are working fine. 

Other thing is, this problem is occurring for any command with play. i.e. play new, play run.

**Comment by xael-fry** [comment]
I am not sure to undestand how you reproduce the probelm. So you have ```~ Oops,
~ Module not found: /opt/play-1.4.4/modules/mongo``` even when running play new ? and play dependencies ?

**Comment by yogesh-oyo** [comment]
Yes, Exactly. Is this some kind of caching?

**Comment by xael-fry** [comment]
Did you run play clean first ? could you give the whole log of play dependencies ?

**Comment by xael-fry** [comment]
@yogesh-oyo  Do you solve your problem?

**Comment by asolntsev** [comment]
@yogesh-oyo Can you create a small project on github that reproduces your problem? Then we could investigate it.
